File: app/src/main/java/com/evaluation/model/result/name/Name.java
package com.evaluation.model.result.name;

import com.google.gson.annotations.SerializedName;

/**
 * @author Vladyslav Havrylenko
 * @since 16.03.2020
 */
public class Name {

    @SerializedName("title")
    private String mTitle;

    @SerializedName("first")
    private String mFirst;

    @SerializedName("last")
    private String mLast;

    public String getTitle() {
        return mTitle;
    }

    public String getFirst() {
        return mFirst;
    }

    public String getLast() {
        return mLast;
    }
}

File: app/src/main/java/com/evaluation/model/result/name/Name.kt
package com.evaluation.model.result.name

import com.google.gson.annotations.SerializedName

/**
 * @author Vladyslav Havrylenko
 * @since 16.03.2020
 */
data class Name(
    @SerializedName("title")
    val mTitle: String,

    @SerializedName("first")
    val mFirst: String,

    @SerializedName("last")
    val mLast: String
)